ci/README.md

```diff
@@ -1,9 +1,15 @@
 ## Overview
 Travis is enabled in this repository so each pull request is checked before being allowed to merge.
-The system is based on fiware/orion-ci:rpm7  which is built from master branch each time a new PR lands in master,
+The system is based on fiware/orion-ci:rpm7 which is built from master branch each time a new PR lands in master,
 providing a clean environment with all build dependencies onboard. The Dockerfile used to build this docker is available
 in the ci/rpm7 directory.
 
+Note that fiware/orion-ci:rpm7 is *not* rebuilt due to changes in the PR branch under test. Thus, if you are developing
```

Comment:
Please  @caa06d9c ensure that what I say in this paragraph is fully correct (or if some modification needs to be done in order to make it 100% accurate).

Reply:
NTC